src/v1/errors/getPathMessage.ts

```diff
@@ -0,0 +1,2 @@
+export const getPathMessage = (path?: string): string =>
```

Comment:
```suggestion
export const getInfoTextForItemPath = (path?: string): string =>
```

I think something like this would read better, not 100% certain that this variation is the best though, wdyt?